airbnb_project/database/airbnb_db_creator.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE IF NOT EXISTS listing (
+    airbnb_listing_id VARCHAR(20) PRIMARY KEY,
```

Comment:
I'd probably increase this to 50. They may not be that long today but could be the case they get longer as time goes on.